static/script/devices/mediaplayer/html5.js

```diff
@@ -62,8 +62,16 @@ require.def(
                     var device = RuntimeContext.getDevice();
 
                     var idSuffix = mediaType === MediaPlayer.TYPE.AUDIO ? "Audio" : "Video";
```

Comment:
This will end up being Video for LIVE_AUDIO? I assume you don't have a test around this otherwise it would have been picked up, so it would be good to add that in.